src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+const bottomLineHeight = 1;
+const topBarHeight = 56;
+
+export const StyledTopBarWrapper = styled.nav`
```

Comment:
[I think it should be `header`](https://www.semrush.com/blog/semantic-html5-guide/)